src/test/java/com/enonic/xp/app/users/rest/resource/AdminResourceTestSupport.java

```diff
@@ -7,6 +7,6 @@
 {
     public AdminResourceTestSupport()
     {
-        super( ResourceConstants.REST_ROOT );
+        super( ResourceConstants.REST_ROOT.substring( 1 ) );
```

Comment:
This looks suspicious. it used to work and suddenly stopped to work with the new code. I believe, the problem is somewhere else.

Reply:
Tests failed after you added a leading slash to the resource path so had to mitigate it :)